api.planx.uk/helpers.ts

```diff
@@ -204,6 +205,72 @@ const getMostRecentPublishedFlowVersion = async (
   return mostRecent;
 };
 
+export const getMostRecentPublishedFlowDate = async (
+  id: string,
+): Promise<string | undefined> => {
+  const { flow } = await $public.client.request<PublishedFlowsResponse>(
+    gql`
+      query GetMostRecentPublishedFlowVersion($id: uuid!) {
+        flow: flows_by_pk(id: $id) {
+          publishedFlows: published_flows(
+            limit: 1
+            order_by: { created_at: desc }
+          ) {
+            created_at
+          }
+        }
+      }
+    `,
+    { id },
+  );
+
+  const mostRecent = flow?.publishedFlows?.[0]?.created_at;
+  return mostRecent;
+};
+
+export interface FlowHistoryEntry {
+  id: number;
+  createdAt: string;
+  firstName: string;
+  lastName: string;
+  type: "comment" | "operation";
+  comment: string | null;
+  data: any; // Operation["data"] via editor's sharedb json OT types
```

Comment:
Maybe we'll eventually move all our ShareDB types to planx-core, but I'm just defaulting to an `any` here for now with comment since it's a read-only response type - it's _correctly_ typed on the frontend where we're actually handling writes/mutations :ok_hand: